imagedephi/redact/svs.py

```diff
@@ -187,11 +188,29 @@ def report_missing_rules(self, report=None) -> None:
                     if report is not None:
                         report[self.image_path.name]["missing_keys"].append(key)
 
-    def report_plan(self) -> dict[str, dict[str | int, str | int]]:
+    def report_plan(
+        self,
+    ) -> dict[
+        str,
+        dict[
+            str,
+            int
+            | str
+            | dict[str, str | int | float | bytes | list[int | float] | dict[str, str | int]],
+        ],
+    ]:
```

Comment:
Writing this out in multiple places seems very awkward to me. I tried to do to make a generic type that would be reusable but it looks like what I wanted to do is only available in [3.12](https://docs.python.org/3/whatsnew/3.12.html#whatsnew312-pep695) so I don't know if that is advisable.
@manthey Do you have a recomendation?

Reply:
Can't you just define a type once and reuse it?  That is, can't you do
```
PlanType = dict[
    str,
    dict[
        str,
        int | str | dict[str, str | int | float | bytes | list[int | float] | dict[str, str | int]],
    ],
]
```
and then use `-> PlanType` for this method?

Reply:
updated in 488d7e61674124925ac40b04c55a1449719228eb